src/query/anonymization.c

```diff
@@ -599,17 +603,17 @@ static void wrap_having_qual(Query *query)
       COERCE_EXPLICIT_CALL);
 }
 
-static void compile_anonymizing_query(Query *query, List *personal_relations, AnonQueryLinks *anon_links)
+static void compile_anonymizing_query(Query *query, List *personal_relations, AnonQueryLinks *anon_links, ParamListInfo bound_params)
```

Comment:
We are dragging this everywhere. Should we bundle in a struct alongside the query?

Reply:
Yeah, this crossed my mind as well, but I'm not 100% sold. What would we call this bundle and is this an abstraction? `PgDiffixQuery`? `AnonymizableQuery`? hmmm...

Reply:
Let's not do anything until we have a stronger argument.